src/Triangle.java

```diff
@@ -0,0 +1,67 @@
+
+import java.util.ArrayList;
+
+public class Triangle extends Polygon {
+
+	public Triangle(int numOfSides, ArrayList<Point> coors) { // 2 i
+		super(numOfSides, coors);
+	}
+
+	public Triangle(int numOfSides) { // 2 ii
+		super(numOfSides);
+	}
+
+	public Triangle(ArrayList<Point> coors) { // 3 i
+		super(3, coors);
+
+	}
+
+	
+	public Triangle() { // 3 ii
+		numOfSides = 3;
+	}
+	/*
```

Comment:
This is not really what I expected but it workes. do `super(3)` then numOfSides would be automatically set to 3 by calling Line 12-16 in the Polygon class.